tools/converter/mule2ballerina/src/main/java/org/wso2/ei/tools/mule2ballerina/visitor/TreeVisitor.java

```diff
@@ -330,4 +367,12 @@ private void createVariableWithEmptyMap(String typeOfTheParamater, String variab
         ballerinaASTAPI.createVariable(variableName, exprAvailable);
         refVarName = variableName;
     }
+
+    private String createVariableOfTypeString(String value, boolean exprAvailable) {
+        ballerinaASTAPI.addTypes(Constant.BLANG_TYPE_STRING); //type of the variable
+        ballerinaASTAPI.createStringLiteral(value);
+        String variableName = Constant.BLANG_VAR_STRING_PAYLOAD + ++variableCounter;
+        ballerinaASTAPI.createVariable(variableName, exprAvailable); //name of the variable
```

Comment:
Shall we enclose the counter ?

Reply:
this is fixed in here >> https://github.com/wso2/product-ei/commit/0ed7662a6af11881f37ec6c941db1abbd454f86f